repo/packages/M/marathon/0/config.json

```diff
@@ -0,0 +1,378 @@
+{
+  "additionalProperties" : false,
+  "description" : "Marathon universe properties",
+  "properties" : {
+    "marathon" : {
+      "additionalProperties" : false,
+      "description" : "Marathon app configuration properties.",
+      "properties" : {
+        "id" : {
+          "description" : "The id of this marathon application name.",
+          "type" : "string"
+        },
+        "cpus" : {
+          "default" : 2,
+          "description" : "CPU shares to allocate to each Marathon instance.",
+          "minimum" : 0,
+          "type" : "number"
+        },
+        "mem" : {
+          "default" : 1024,
+          "description" : "Memory (MB) to allocate to each Marathon task.",
+          "minimum" : 512,
+          "type" : "number"
+        },
+        "instances" : {
+          "default" : 1,
+          "description" : "Number of Marathon instances to run.",
+          "minimum" : 0,
+          "type" : "integer"
+        },
+        "uris" : {
+          "default" : [ ],
+          "description" : "List of URIs that will be download and made available in the current working directory of Marathon. For example this can be used to download a Java keystore file for SSL configuration.",
+          "items" : {
+            "pattern" : "^[\\s]+",
+            "type" : "string"
+          },
+          "type" : "array"
+        }
+      },
+      "required" : [ "cpus", "mem", "instances" ],
+      "type" : "object"
+    },
+    "jvm" : {
+      "additionalProperties" : false,
+      "description" : "JVM configuration properties",
+      "properties" : {
+        "heap-min" : {
+          "default" : 256,
+          "description" : "Memory (MB) start size for the JVM heap. This number should be be less or equals than the jvm-heap-max.",
+          "minimum" : 0,
+          "type" : "integer"
+        },
+        "heap-max" : {
+          "default" : 768,
+          "description" : "Memory (MB) max size for the JVM heap. This number should be be less than the absolute memory.",
+          "minimum" : 0,
+          "type" : "integer"
+        }
+      },
+      "required" : [ "heap-min", "heap-max" ],
+      "type" : "object"
+    },
+    "commandLine" : {
+      "additionalProperties" : true,
+      "description" : "Marathon command line flags. These are the same flags that are passed through to Marathon when launching manually from the command line. See details here: https://mesosphere.github.io/marathon/docs/command-line-flags.html",
+      "properties" : {
+        "access-control-allow-origin" : {
+          "description" : "The origin(s) to allow in Marathon. Not set by default. Example values are \"*\", or \"http://localhost:8888, http://domain.com\"",
+          "type" : "string"
+        },
+        "artifact-store" : {
+          "description" : "URL to the artifact store. Supported store types hdfs, file. Example: hdfs://localhost:54310/path/to/store, file:///var/log/store",
```

Comment:
The supported store types define the protocol of the URL. Thats why it should be lower case.